pipeline/dags/probe_api.py

```diff
@@ -0,0 +1,126 @@
+import pendulum
+
+from airflow.decorators import dag, task
+from airflow.operators import empty
+
+from dag_utils.virtualenvs import PYTHON_BIN_PATH
+
+default_args = {}
+
+
+@task.external_python(
+    python=str(PYTHON_BIN_PATH),
+)
+def store_probe_results(get_requests: list[str], today: str) -> None:
+    import json
+
+    import furl
+    import requests
+
+    from airflow.models import Variable
+
+    from dag_utils import s3
+
+    BASE_URI = "https://api.data.inclusion.gouv.fr"
+    token_probe = Variable.get("DATA_INCLUSION_API_PROBE_TOKEN")
+    headers = {
+        "Authorization": f"Bearer {token_probe}",
+    }
+    for get_request in get_requests:
+        url = furl.furl(f"{BASE_URI}/{get_request}")
+
+        response = requests.get(url.add({"size": 100}).url, headers=headers)
+        param_slug = [f"{k}-{v}" for k, v in url.query.params.items()]
+        file_name = f"tests/{today}/{'-'.join(list(url.path.segments) + param_slug)}"
+        s3.store_content(
+            f"tests/{today}/{file_name}.json",
+            json.dumps(response.json(), indent=2).encode(),
+        )
+
+
+@task.external_python(
+    python=str(PYTHON_BIN_PATH),
+)
+def compare_results(get_requests: list[str], today: str, yesterday: str) -> None:
+    import json
+
+    import furl
+    from botocore.exceptions import ClientError
+
+    from dag_utils import s3
+
+    error_margin = 0.1
```

Comment:
proposition : `# 10% like the threshold for source stats alerts. Adjust if needed.`